R/tm_g_bivariate.R

```diff
@@ -363,7 +381,52 @@ srv_g_bivariate <- function(id,
       color = color, fill = fill, size = size
     )
 
-    selector_list <- teal.transform::data_extract_multiple_srv(data_extract, data)
+    rule_var <- function(other) {
+      function(value) {
+        othervalue <- selector_list()[[other]]()$select
+        if (length(value) == 0L && length(othervalue) == 0L)
+          "Please select at least one of x-variable or y-variable"
+      }
+    }
+    rule_diff <- function(other) {
```

Comment:
So if you only include row_facet and not column_facet as an argument to the module you get an error:
![image](https://user-images.githubusercontent.com/15201933/210366556-2a3e4945-d7e1-476e-9edb-19af0cc3c63f.png)